### ISSUE TITLE
Support overriding shorthand properties

### ISSUE THREAD

**Issue by btpoe** [ben-eb/postcss-merge-longhand#22]
It would be great to be able to write:

``` css
a {
  padding: 3px;
  padding-left: 5px;
}
```

and have it condense to:

``` css
a {
  padding: 3px 3px 3px 5px;
}
```

Currently, the lib only supports merging if all longhand properties are declared.

**Comment by ben-eb** [comment]
Agreed. Some of this is already done in the refactor branch, but it doesn't yet provide optimal results.

https://github.com/ben-eb/postcss-merge-longhand/blob/refactor/src/__tests__/index.js#L111-L113

**Comment by ben-eb** [comment]
Resolved in [3.0](https://github.com/ben-eb/postcss-merge-longhand/commit/947185b1b19549410f943e2c24184e809746e231).
